specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1007 @@
+  <tr>
+    <td><code>delivery</code></td>
+    <td><a href="#deliveryspec">DeliverySpec</a><br/>(OPTIONAL)</td>
+    <td>Delivery options for this Subscription.</td>
+    <td>RECOMMENDED</td>
+  </tr>
+</table>
+
+#### Status
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>conditions</code></td>
+    <td><a href="#error-signalling">See Error Signalling</a></td>
+    <td>Used for signalling errors, see link.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>observedGeneration</code></td>
+    <td>int64</td>
+    <td>The latest metadata.generation that the reconciler has attempted. If <code>observedGeneration</code> is updated, <code>conditions</code> MUST be updated with current status in the same transaction.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>physicalSubscription</code></td>
+    <td><a href="#physicalsubscriptionstatus">PhysicalSubscriptionStatus</a></td>
+    <td>The fully resolved values for <code>spec</code> endpoint references.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### Addressable
+
+Note that the Addressable interface is a partial schema -- any resource which
+includes these fields MAY be referenced using a `duckv1.Destination`.
+
+#### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource. Note that there are no restrictions on `apiVersion` or `kind` as long
+as the object matches the partial schema.
+
+#### Spec:
+
+There are no `spec` requirements for Addressable.
+
+#### Status
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>address</code></td>
+    <td><a href="#duckv1addressable">duckv1.Addressable</a></td>
+    <td>Address used to deliver events to the resource.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+## Detailed SubResource Objects
+
+### duckv1.Addressable
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>url</code></td>
+    <td>URL (string)</td>
+    <td>Address used to deliver events to the Addressable.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### duckv1.Destination
+
+Destination is used to indicate the destination for event delivery. A
+Destination eventually resolves the supplied information to a URL by resolving
+`uri` relative to the address of `ref` (if provided) as described in
+[Addressable resolution](#addressable-resolution). `ref` MAY be an
+[Addressable](#duckv1-addressable) object or a `v1/Service` Kubernetes service.
```

Comment:
Let's add some text here too about how other mechanisms for resolving a ref to a URI MAY be supported